Sources/AppcuesKit/Data/Models/Experience.swift

```diff
@@ -45,6 +45,7 @@ internal struct FailedExperience: Decodable {
             name: name ?? "",
             type: type ?? "",
             publishedAt: publishedAt,
+            context: nil,
```

Comment:
on Android side, I added the optional `context` on the failed version as well, since it may lend some insight into the experience error that gets generated - nice to know which localized version it came from, if available. I think harmless to try, if optional on the deserialization.

Reply:
Good idea. Updated in 12c7b70.